src/geo/ui/legends/base/legend-view-base.js

```diff
@@ -21,41 +19,35 @@ var LegendViewBase = Backbone.View.extend({
 
     this._loadImages();
 
-    if (this.model.isVisible()) {
-      this.$el.show();
-    } else {
-      this.$el.hide();
-    }
+    this.model.isVisible()
+      ? this.$el.show()
```

Comment:
😍

Reply:
Or `this.$el.toggle(this.model.isVisible())` but I prefer the way you implemented.

Reply:
@nobuti comment is the winner, IMHO 💃 .